docs/index.rst

```diff
@@ -6,18 +6,37 @@
 Welcome to MDAkit MembCurv's documentation!
 =========================================================
 
+MDAnalysis membrane curvature analysis module calculates gaussian and mean 
```

Comment:
```suggestion
The MDAnalysis membrane curvature analysis module calculates the Gaussian and mean 
```

I think Gaussian is typically capitalized

Reply:
Thanks. Changed.